.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @erikvansebille @veckothegecko @willirath
\ No newline at end of file
+* @erikvansebille @VeckoTheGecko @willirath
```

Comment:
I see an error message on this file:

Unknown owner on line 1: make sure @VeckoTheGecko exists and has write access to the repository
* @erikvansebille @VeckoTheGecko @willirath

Do you know how/where I should add you, @VeckoTheGecko ?

Reply:
I think we have to use the bot https://github.com/conda-forge/parcels-feedstock/pull/119